src/ci/docker/host-aarch64/dist-aarch64-linux/build-clang.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
```

Comment:
Could you please move this script (from here and `dist-x86_64-linux`) to `src/ci/scripts` and parametrize `DLLVM_TARGETS_TO_BUILD` through an environment variable? It's annoying enough to keep the LLVM versions in sync without them being in yet another bash script :)

Same for the GCC/ZSTD build script. And also the `shared.sh` file, it's unnecessarily duplicated here, in `x86_64` and other `dist-XYZ` builderfolders.

Thanks.

Reply:
While I'm at it, should I just remove `shared.sh` to `hide-build-output.sh`? Should be clearer as to what it does

Reply:
Oh okay this is a bit of a mess. We've got:
- `ci/shared.sh` - a bunch of helpers
- `docker/scripts/shared.sh` - hide_output and retry
- `docker/host-x86_64/dist-x86_64-linux/shared.sh` - slightly different hide_output

Reply:
Then I think maybe remove `docker/host-x86_64/dist-x86_64-linux/shared.sh` and just source the one in `docker/scripts/shared.sh`? The functionality should be the same